src/main/java/com/zontik/groshiky/format/DateFormat.java

```diff
@@ -0,0 +1,5 @@
+package com.zontik.groshiky.format;
+
+public class DateFormat {
+    public static final String FORMATTER = "dd MM yyyy - HH:mm";
```

Comment:
This is not Formatter..., probably more appropriate name would be DateFormatString, or DateFormat